plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConsumerFactory.java

```diff
@@ -15,14 +15,18 @@
 package io.trino.plugin.kafka;
 
 import io.trino.spi.connector.ConnectorSession;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.clients.consumer.KafkaConsumer;
 
 import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public interface KafkaConsumerFactory
 {
     default KafkaConsumer<byte[], byte[]> create(ConnectorSession session)
     {
+        Logger.getLogger(ConsumerConfig.class.getName()).setLevel(Level.WARNING);
```

Comment:
We cannot do it like that.

First it would have to be done once per connector initialization so somewhere in the `io.trino.plugin.kafka.KafkaConnectorFactory#create`.
Also if set logging statically then you would override anything that is set via https://trino.io/docs/current/admin/properties-logging.html.

So I think we need to disable logging in tests. Somewhere here `io.trino.plugin.kafka.KafkaQueryRunnerBuilder#build` and update docs to mention that it is a good practice to set this in `log.properties` so your log file is less polluted.

I would do it as two PRs as they require different people for review.

Reply:
> Also if set logging statically then you would override anything that is set via https://trino.io/docs/current/admin/properties-logging.html.
Yeah I was looking for alternatives like a connector level conf file but can't find any. Didn't want to change the default `log.properties`.

So instead of hard coding it here, we just have to document how to suppress this & hard code `setLevel` in tests, right?

Reply:
Also looks like Kafka logging is already suppressed in [KafkaQueryRunnerBuilder](https://github.com/trinodb/trino/blob/master/plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/KafkaQueryRunnerBuilder.java#L64). So this should be a doc change only?

Reply:
> So this should be a doc change only?

Yes, it looks like that. There also product tests where we can implement this suggestion. So we can avoid poluting logs in product tests.
See `io.trino.tests.product.launcher.env.common.Kafka` where we could have log.properties file that is used.

Please reach to @andrewdibiasio6 or @lukasz-walkiewicz to help you run product tests for kafka here.

Reply:
- Lower log level in product tests: https://github.com/trinodb/trino/pull/14149
- Document log level: https://github.com/trinodb/trino/pull/14150